DataFormats/Math/interface/GeantUnits.h

```diff
@@ -27,36 +27,36 @@ namespace geant_units {
     using angle_units::operators::convertRadToDeg;
 
     // Length
-    constexpr long double operator"" _mm(long double length) { return length * 1.; }
-    constexpr long double operator"" _cm(long double length) { return length * 10.; }
-    constexpr long double operator"" _m(long double length) { return length * 1000.; }
-    constexpr long double operator"" _cm3(long double length) { return length * 1._cm * 1._cm * 1._cm; }
-    constexpr long double operator"" _m3(long double length) { return length * 1._m * 1._m * 1._m; }
-    constexpr long double operator"" _mm(unsigned long long int length) { return length * 1; }
-    constexpr long double operator"" _cm(unsigned long long int length) { return length * 10; }
+    constexpr double operator"" _mm(long double length) { return length * 1.; }
+    constexpr double operator"" _cm(long double length) { return length * 10.; }
```

Comment:
@ianna , do we need arguments "long double length"?

Reply:
Yes. Only the following parameter lists are allowed on literal operators :

( const char * ) | (1) |  
( unsigned long long int ) | (2) |  
( long double ) | (3) |  
( char ) | (4) |  
( wchar_t ) | (5) |  
( char8_t ) | (6) | (since C++20)
( char16_t ) | (7) |  
( char32_t ) | (8) |   
( const char * , std::size_t ) | (9) |   
( const wchar_t * , std::size_t ) | (10) |  
( const char8_t * , std::size_t ) | (11) | (since C++20)
( const char16_t * , std::size_t ) | (12) |   
( const char32_t * , std::size_t ) | (13)

Reply:
operator"" supports only "long" arguments